apps/mobile/src/components/OfferForm/components/Expiration/index.tsx

```diff
@@ -22,39 +26,85 @@ function Expiration({
   const {t} = useTranslation()
   const locale = getCurrentLocale()
   const [expirationDate, setExpirationDate] = useAtom(expirationDateAtom)
+  const [expirationDateActive, setExpirationDateActive] = useState(
+    !!expirationDate
+  )
```

Comment:
Can this be derived from exirationDate? 

```suggestion
 const expirationDateActive === !!expirationDate
```